src/DependencyInjection/Compiler/AddResourcesPathsPass.php

```diff
@@ -41,7 +41,7 @@ private function getResourcesPath(ContainerBuilder $container)
         $rootDir = dirname($container->getParameter('kernel.root_dir'));
 
         foreach ($container->getParameter('kernel.bundles') as $name => $class) {
-            if ('Contao\CoreBundle\HttpKernel\Bundle\ContaoModuleBundle' === $class) {
+            if (is_a($class, 'Contao\CoreBundle\HttpKernel\Bundle\ContaoModuleBundle', true)) {
```

Comment:
`is_a()` is deprecated in favor of the `instanceof` operator and will trigger an `E_STRICT` warning.